var/spack/repos/builtin/packages/gperftools/package.py

```diff
@@ -3,38 +3,76 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import os
 from spack.package import *
 
 
-class Gperftools(AutotoolsPackage):
+class Gperftools(CMakePackage):
     """Google's fast malloc/free implementation, especially for
        multi-threaded applications.  Contains tcmalloc, heap-checker,
        heap-profiler, and cpu-profiler.
 
     """
     homepage = "https://github.com/gperftools/gperftools"
-    url      = "https://github.com/gperftools/gperftools/releases/download/gperftools-2.7/gperftools-2.7.tar.gz"
-    maintainers = ['albestro', 'eschnett', 'msimberg', 'teonnik']
+    url = "https://github.com/gperftools/gperftools/archive/refs/tags/gperftools-0.0.tar.gz"
+    maintainers = ["albestro", "eschnett", "msimberg", "teonnik"]
 
-    version('2.9.1', sha256='ea566e528605befb830671e359118c2da718f721c27225cbbc93858c7520fee3')
-    version('2.8.1', sha256='12f07a8ba447f12a3ae15e6e3a6ad74de35163b787c0c7b76288d7395f2f74e0')
-    version('2.7', sha256='1ee8c8699a0eff6b6a203e59b43330536b22bbcbe6448f54c7091e5efb0763c9')
-    version('2.4', sha256='982a37226eb42f40714e26b8076815d5ea677a422fb52ff8bfca3704d9c30a2d')
-    version('2.3', sha256='093452ad45d639093c144b4ec732a3417e8ee1f3744f2b0f8d45c996223385ce')
+    version("2.10", sha256="b0dcfe3aca1a8355955f4b415ede43530e3bb91953b6ffdd75c45891070fe0f1")
+    version("2.9.1", sha256="484a88279d2fa5753d7e9dea5f86954b64975f20e796a6ffaf2f3426a674a06a")
+    version("2.8.1", sha256="260c510b742e44bc53465a1e9b3294f290525360658a6d1612019df2c2f7f307")
+    version("2.7", sha256="1ee8c8699a0eff6b6a203e59b43330536b22bbcbe6448f54c7091e5efb0763c9", deprecated=True)
```

Comment:
Why are you deprecating these versions? Are they simply old, or is there a specific reason, e.g. a security issue?

Reply:
Because they use Autotools and I'd like to eventually get rid of this double setup supporting both CMake and Autotools.